public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -99,7 +99,9 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
       }));
 
     for (const target of filteredTargets) {
-      if (options.app === CoreApp.Explore) {
+      // In explore we want to show instant query in a table under the graph panel to mimic behaviour of prometheus.
+      // We don't want to do that in dashboards though as user would have to pick the correct data frame.
```

Comment:
```suggestion
      // In explore we want to show result of metrics instant query in a table under the graph panel to mimic behaviour of prometheus.
      // We don't want to do that in dashboards though as user would have to pick the correct data frame.
```

Just a nit suggestion